packages/styles/icon-button.css

```diff
@@ -1,42 +1,42 @@
 .IconButton {
+  position: relative;
   display: inline-flex;
   justify-content: center;
   align-content: center;
   height: var(--button-thin-height);
   width: var(--button-thin-height);
-  border: 1px solid transparent;
   border-radius: 3px;
+  border: none;
   background-color: var(--icon-button-background-color);
   color: var(--icon-button-icon-color);
   margin: 2px;
+  padding: 0;
+  --button-hover-outline-color: var(--top-bar-text-color);
```

Comment:
This looks like a typo. I don't think `--button-hover-outline-color` is a valid CSS property.

Reply:
Wasn't a typo, it's a custom key that we were using for the hover outline. I essentially stole this methodology from the vanilla button css. Since it isn't repeated, I'll remove it for now.

Reply:
CSS is crazy 🤯